### PR TITLE
i18n: import stack pack translations

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
Importing new i18n strings as they come in.  In this import: stack packs.
image

SEO groups will be imported in another PR, as it is separate in tc/

### CODE REVIEW

File: lighthouse-core/test/lib/i18n/i18n-test.js [hunk 1]
```diff
@@ -59,7 +59,6 @@ describe('i18n', () => {
     it('returns icu messages in the specified locale', () => {
       const strings = i18n.getRendererFormattedStrings('en-XA');
       expect(strings.passedAuditsGroupTitle).toEqual('[Þåššéð åûðîţš one two]');
-      expect(strings.scorescaleLabel).toEqual('[Šçöŕé šçåļé: one two]');
```
Comment (brendankenny) [inline lighthouse-core/test/lib/i18n/i18n-test.js]: can you replace this rather than removing it

Comment (exterkamp) [reply lighthouse-core/test/lib/i18n/i18n-test.js]: Done.

**Comment by exterkamp** [comment]
> > SEO groups will be imported in another PR, as it is separate in tc/
> 
> seems like this isn't true? Are they all in here?

They are all in here in English as only lang for all langs (artifact of tc/ since they are being translated they appear as English in all langs).  They just aren't translated yet.  That's the follow up.

**Comment by brendankenny** [comment]
> They just aren't translated yet

oh, I see

**Pull request merged**